style.css

```diff
@@ -0,0 +1,466 @@
+.header-background {
+  position: absolute;
+  width: 475px;
+  height: 916px;
+  left: 0;
+  top: 1px;
+
+  /* Neutrals/N0 */
+
+  background-color: #fff;
+}
+
+.dkl {
+  float: right;
+  margin-right: 90px;
+}
+
```

Comment:
- [ ] The mobile version of the portfolio should be closed to the original design Check the [templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) Kindly follow the original vision, you have to use the same font, colors (background and font) and the same margins (top, bottom, left, right) defined in the design if it's pretty and works, it will be a wonderful website. 🎨 🖌️ 

- Your design:

![17bad](https://user-images.githubusercontent.com/79058364/185247701-056dfd4b-1b32-4873-b42c-71ba7d57fdc6.png)

- [templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)

![17god](https://user-images.githubusercontent.com/79058364/185247776-bdb2d956-f2e9-44b6-b6c6-03d3f543bf77.png)


NOTE: Minimum mobile screen size should be `375`px. Otherwise, the mobile version should be responsive until the desktop screen size.